### DISCUSSION TITLE
Can the Throwable's cause(s) be added to the structured JSON logging that was just added in LOGBACK-1734?

### DISCUSSION THREAD

**Discussion by cpholt** [qos-ch/logback#673] [General]
LOGBACK-1734 is great!!  Thank you!!  It connects logback output very easily now to fluentbit.

Can the Throwable's cause(s) be added to the structured JSON logging that was just added in LOGBACK-1734?  I think it would be relatively easy to implement in ch.qos.logback.classic.encoder.JsonEncoder.  Perhaps add a "cause" member to the "throwable" item, where the cause can also itself have a cause?

**Comment by ceki** [comment]
@cpholt  Thank you for the encouraging feedback. 

Please create a jira issue at jira.qos.ch asking for this issue to be fixed.

**Comment by cpholt** [comment]
Ceki,

I see this resolved in the upcoming 1.3.9 release.  Is there a planned date
for that?  I can build a 1.3.9-SNAPSHOT version and incorporate that, but
just curious on when the official 1.3.9 might be published.

Thanks again for such a great piece of work!
